packages/@uppy/react/src/getHTMLProps.js

```diff
@@ -1,164 +1,3 @@
-// Attributes in the format of htmlAtrribute: reactAttribute
-const possibleStandardNames = [
-  'accept',
-  'acceptCharset',
-  'acceptCharset',
-  'accessKey',
-  'action',
-  'allowFullScreen',
-  'alt',
-  'as',
-  'async',
-  'autoCapitalize',
-  'autoComplete',
-  'autoCorrect',
-  'autoFocus',
-  'autoPlay',
-  'autoSave',
-  'capture',
-  'cellPadding',
-  'cellSpacing',
-  'challenge',
-  'charSet',
-  'checked',
-  'children',
-  'cite',
-  'className',
-  'classID',
-  'className',
-  'cols',
-  'colSpan',
-  'content',
-  'contentEditable',
-  'contextMenu',
-  'controls',
-  'controlsList',
-  'coords',
-  'crossOrigin',
-  'dangerouslySetInnerHTML',
-  'data',
-  'dateTime',
-  'default',
-  'defaultChecked',
-  'defaultValue',
-  'defer',
-  'dir',
-  'disabled',
-  'disablePictureInPicture',
-  'disableRemotePlayback',
-  'download',
-  'draggable',
-  'encType',
-  'enterKeyHint',
-  'htmlFor',
-  'form',
-  'formMethod',
-  'formAction',
-  'formEncType',
-  'formNoValidate',
-  'formTarget',
-  'frameBorder',
-  'headers',
-  // 'height',
-  'hidden',
-  'high',
-  'href',
-  'hrefLang',
-  'htmlFor',
-  'httpEquiv',
-  'httpEquiv',
-  'icon',
-  'id',
-  'innerHTML',
-  'inputMode',
-  'integrity',
-  'is',
-  'itemID',
-  'itemProp',
-  'itemRef',
-  'itemScope',
-  'itemType',
-  'keyParams',
-  'keyType',
-  'kind',
-  'label',
-  'lang',
-  'list',
-  'loop',
-  'low',
-  'manifest',
-  'marginWidth',
-  'marginHeight',
-  'max',
-  'maxLength',
-  'media',
-  'mediaGroup',
-  'method',
-  'min',
-  'minLength',
-  'multiple',
-  'muted',
-  'name',
-  'noModule',
-  'nonce',
-  'noValidate',
-  'open',
-  'optimum',
-  'pattern',
-  'placeholder',
-  'playsInline',
-  'poster',
-  'preload',
-  'profile',
-  'radioGroup',
-  'readOnly',
-  'referrerPolicy',
-  'rel',
-  'required',
-  'reversed',
-  'role',
-  'rows',
-  'rowSpan',
-  'sandbox',
-  'scope',
-  'scoped',
-  'scrolling',
-  'seamless',
-  'selected',
-  'shape',
-  'size',
-  'sizes',
-  'span',
-  'spellCheck',
-  'src',
-  'srcDoc',
-  'srcLang',
-  'srcSet',
-  'start',
-  'step',
-  'style',
-  'summary',
-  'tabIndex',
-  // 'target',
-  'title',
-  'type',
-  'useMap',
-  'value',
-  // 'width',
-  'wmode',
-  'wrap',
-]
```

Comment:
Why is the list no longer needed?

Reply:
I'm hoping we can test `HTMLElement.prototype` instead of having to maintain a list of all possible HTML prop. I haven't tested it yet though, so it may be actually needed.

Reply:
That's interesting! I didn't know that existed.